.travis/setup-kubernetes.sh

```diff
@@ -27,6 +27,7 @@ function install_helm {
     curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
     chmod 700 get_helm.sh
     sudo ./get_helm.sh --version ${TEST_HELM_VERSION}
+    helm init
```

Comment:
Wasn't Helm Init done also in the ST directly?

Reply:
It is. It's necessary to have it before we try to do the build (and it failed without this). OTOH developers might not initialize helm before running the tests (and they are very unlikely to run this script if they're in that position), so I think it's harmless having it in both places. Or maybe we should make it part of the main `Makefile`?

Reply:
Now you got me confused ... does `helm template` need `tiller` to run? Or is it needed for system tests only? If `helm template` needs it, it sucks quite a lot ... because it means you need running cluster just for `make docker_build`. If it is just for system tests its fine to keep it in this `setup_kubernetes.sh` script.

Reply:
`helm template` does not need `tiller` running. But without the `helm init` it fails (like this https://api.travis-ci.org/v3/job/410862480/log.txt, which is a really weird way to fail).

Reply:
But `helm init` needs a running cluster, right? That means that it will break completely the developer workflow. You will need to run the cluster everytime you want to build and push the images.

Is `helm init --client-only` sufficient for this? That will init helm without installing tiller and doesn't need running cluster. So if we add that to make everyone will need to install helm, but at least it will not force the cluster.